modules/gitops-release-notes-1-17-0.adoc

```diff
@@ -0,0 +1,170 @@
+
+[id="removed-support-for-deprecated-initialRepositories-repositoryCredentials-fields_{context}"]
+=== Removed support for deprecated `initialRepositories` and `repositoryCredentials` fields
+
+* In {gitops-title} 1.17, support for the deprecated `initialRepositories` and `repositoryCredentials` fields has been removed. If these fields are used, Argo CD logs a warning indicating their removal. Users can add repositories and repository credentials manually using the Argo CD CLI or the Argo CD dashboard. link:https://issues.redhat.com/browse/GITOPS-5961[GITOPS-5961]
+
+[id="breaking-changes-1-17-0_{context}"]
+== Breaking changes
+
+[id="upgrade-to-argo-cd-3-0_{context}"]
+=== Upgrade to Argo CD 3.0
+
+This release introduces an upgrade to the Argo CD component, transitioning from v2.14 to v3.0. This is a minor upgrade that introduces a list of breaking changes in Argo CD v3.0, with remediation steps provided in the following section. For more information, see link:https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/2.14-3.0/[Upgrading Argo CD from v2.0 to v3.0].
+
+[id="rbac-behavior-changes-with-argo-cd-dex-sso_{context}"]
+=== RBAC behavior changes with Argo CD 3.0 and Dex SSO
+
+* Argo CD 3.0 introduces a change in RBAC handling when using Dex for single sign-on (SSO). The RBAC system uses the `federated_claims.user_id` field instead of the previously encoded sub claim. This change affects how user identities are evaluated and matched in RBAC policies.
++
+To address this, the Argo CD Operator documentation has been updated with:
+
+** Detailed migration guidance for transitioning from sub to federated_claims.user_id
+** Step-by-step remediation procedures
+** Practical RBAC policy examples
+** Best practices for managing RBAC with the new hands-off approach
++
+Users must review and update their RBAC policies to ensure compatibility with Argo CD 3.0 when using the {gitops-title} Operator. For more information, see link:https://argocd-operator.readthedocs.io/en/latest/upgrading/#rbac-with-dex-sso-authentication[Configuring RBAC with Dex SSO Authentication].
++
+--
+link:https://issues.redhat.com/browse/GITOPS-6887[GITOPS-6887]
+--
+
+[id="added-guidance-for-logs-rbac-enforcement-in-argocd_{context}"]
+=== Added guidance for logs RBAC enforcement in Argo CD 3.0
+
+* Before this update, the Argo CD Operator upstream documentation did not explain the changes to logs RBAC introduced in Argo CD 3.0. In contrast to Argo CD 2.x, where users with access to Applications could also view logs, Argo CD 3.0 treats logs as a distinct RBAC resource requiring explicit permissions. With this update, the Argo CD Operator upstream documentation includes:
```

Comment:
Maybe refer to the upstream doc
https://argocd-operator.readthedocs.io/en/latest/upgrading/#logs-rbac-enforcement

cc: @anandf